integration_tests/emukit/bayesian_optimization/test_constrained_loop.py

```diff
@@ -8,6 +8,7 @@
 
 
 def test_optimization_with_linear_constraint():
+    np.random.seed(0)
```

Comment:
Nice! Can you just leave a comment before this line, explaining why this, arguably random, line is here? Something like "Stops the flakiness, see [link] for details" would suffice.

Reply:
done